### PR TITLE
Update mirrorlist

### PR DESCRIPTION
**Pull request by a0xz**

### CODE REVIEW

File: mirrorlist [hunk 1]
```diff
@@ -92,6 +96,8 @@ Server = https://uk-mirror.chaotic.cx/$repo/$arch
 # Arizona
 # * By: ayulauffer <t.me/ayulauffer>
 Server = https://us-az-mirror.chaotic.cx/$repo/$arch
+# Florida
```
Comment (alissonlauffer) [inline mirrorlist:99]:
```suggestion
# Florida
# * By: Alisson <t.me/alissonlauffer>
```

Comment (alissonlauffer) [reply mirrorlist:99]: Also worth changing line 15 to `# * By: Alisson <t.me/alissonlauffer> (São Paulo)` too.

Comment (a0xz) [reply mirrorlist:99]: ok

**Comment by a0xz** [comment]
@dr460nf1r3 update the mirrors

**Review by a0xz — COMMENTED** [review]
ok!

**Comment by dr460nf1r3** [comment]
So it can be merged?

**Pull request closed without merging**